xlators/mount/fuse/src/fuse-helpers.c

```diff
@@ -143,7 +143,6 @@ get_fuse_state (xlator_t *this, fuse_in_header_t *finh)
 }
 
 
-#define FUSE_MAX_AUX_GROUPS 32 /* We can get only up to 32 aux groups from /proc */
```

Comment:
It would be needed to check if we're running on Linux and if the kernel version is 3.8 or higher. This flag/boolean can then be set in the `fuse_private_t` structure which can then be used in `frame_fill_groups`.